src/lib/features/like-episode/ui/like-button.svelte

```diff
@@ -8,14 +8,17 @@
 
   export let episode: Episode;
 
+  export let size = 6;
+  export let iconSize = 4;
+
   $: liked = $likesStore.has(episode.id);
 </script>
 
 <IconButton
   name={liked ? 'Remove the like' : 'Like this episode'}
-  class={clsx('w-6 h-6', liked && 'text-pink-500')}
+  class={clsx(`w-${size} h-${size}`, liked && 'text-pink-500')}
```

Comment:
suggestion: How about we just expose a `class` prop that will go after the default sizing, overriding it if necessary?

Reply:
And `iconClass` as well, that will just go straight to the icon